bindings/python/examples/how_tos/outputs/unlock_conditions.py

```diff
@@ -83,4 +83,4 @@
 )
 outputs.append(foundry_output)
 
-print(json.dumps(outputs, indent=2))
+print(json.dumps(list(map(lambda o: o.as_dict(), outputs)), indent=2))
```

Comment:
This is so cryptic lol, honestly have no clue what's happening here. Not sure we should put stuff like that in how tos, this is a bit confusing

Reply:
It's iterating over the outputs and calls the `as_dict()` method on them, not sure how else we should do it